### PR TITLE
Identity Verification

### PR DESCRIPTION
**Pull request by iamonuwa**
Added a 
1. Lndr verification screen
2. Lndr verification form
3. Lndr Offline Screen

Reordered the Settle Up screen

### CODE REVIEW

**Comment by willbach** [comment]
I'll be looking at this first thing on Monday morning

**Comment by iamonuwa** [comment]
Thank you @willbach

File: packages/lndr/data.ts [hunk 1]
```diff
@@ -0,0 +1,245 @@
+export default [
```
Comment (willbach) [inline packages/lndr/data.ts]: can you name this file something like `countries.ts`? Also, please add in a field called `code` that is the 3-letter country code

Comment (iamonuwa) [reply packages/lndr/data.ts]: Great! I'll effect the changes.

**Comment by iamonuwa** [comment]
That'll be great. I'll be expecting it soon. Glad you like it BTW. 👍

**Comment by canterberry** [comment]
Is this pull request still a work in progress, or has it been superceded by https://github.com/blockmason/lndr-mobile/pull/372 ?

**Pull request closed without merging**